packages/module-federation/src/utils/parse-static-remotes-config.ts

```diff
@@ -23,9 +23,9 @@ export function parseStaticRemotesConfig(
   const config: Record<string, StaticRemoteConfig> = {};
   for (const app of staticRemotes) {
     const outputPath =
-      context.projectGraph.nodes[app].data.targets['build'].options.outputPath;
-    const basePath = dirname(outputPath);
-    const urlSegment = basename(outputPath);
+      context.projectGraph.nodes[app].data.targets['build'].options.outputPath; // dist || dist/checkout
+    const basePath = outputPath === 'dist' ? outputPath : dirname(outputPath); // dist || dist/checkout -> dist
```

Comment:
If we hardcode `dist` but users change it to something else like `build`, what would be the behavior?

Should we parse dirname, and if it's empty then default to `outputPath`?

Reply:
Ah yes good point. I’ll update this.